arm-relay/2017-04-01/swagger/relay.json

```diff
@@ -1575,9 +1581,6 @@
           "description": "Resource tags"
         }
       },
-      "required": [
```

Comment:
that's the reason I was asking if they are really tracked resources, and if so, I pulled in @ravbhatnagar to comment as it has to do with the api design. @ravbhatnagar could you provide your thoughts on this issue?